horovod/mxnet/__init__.py

```diff
@@ -51,7 +51,8 @@ def create_state_multi_precision(self, index, weight):
     def _do_allreduce(self, index, grad):
         if isinstance(index, (tuple, list)):
             for i in range(len(index)):
-                allreduce_(grad[i], average=False, name=str(index[i]))
+                allreduce_(grad[i], average=False,
+                           name=str(index[i]), priority=-i)
```

Comment:
why the priority is decreasing with index? can't we just all use 0?

Reply:
If all is 0, there is no priority anymore. Followed the way of [pushing](https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/trainer.py#L354) to kvstore.